### DISCUSSION TITLE
Support a `latest` alias to docs for the latest version

### DISCUSSION THREAD

**Discussion by sindresorhus** [SwiftPackageIndex/SwiftPackageIndex-Server#2133] [Ideas]
I know that https://swiftpackageindex.com/sindresorhus/KeyboardShortcuts/documentation links to the latest version, but I use an enum for namespacing, so the landing page is kinda useless. I would like to deep link the docs.

Currently I use the URL https://swiftpackageindex.com/sindresorhus/keyboardshortcuts/main/documentation/keyboardshortcuts/keyboardshortcuts but it links to the `main` branch. It would be nice if I could use a link like this, but replace `main` with `latest` to make it to redirect to that exact deep link, but for the latest version.

**Comment by finestructure** [comment]
You can just drop the `main` from the url: we keep the path after the `documentation` and redirect to the latest. I.e. this link should do exactly what you're looking for:

https://swiftpackageindex.com/sindresorhus/keyboardshortcuts/documentation/keyboardshortcuts/keyboardshortcuts

Does that help?

**Reply by finestructure** [reply]
Another solution is to add an overview article to your docs, like we did here for example:

https://swiftpackageindex.com/SwiftPackageIndex/SPIManifest/0.12.0/documentation/spimanifest

This is generated from this file:

https://github.com/SwiftPackageIndex/SPIManifest/blob/main/Sources/SPIManifest/Documentation.docc/SPIManifest.md

Hope that helps!

**Reply by sindresorhus** [reply]
> Does that help?

That works perfectly. Thank you 👍

**Reply by hassila** [reply]
I just found this before opening a duplicate- super it’s possible!
